lib/classes/ConfigSchemaHandler/normalizeAjvErrors.js

```diff
@@ -238,6 +238,17 @@ const improveMessages = (resultErrorsSet) => {
   }
 };
 
+const filterDuplicateErrors = (resultErrorsSet) => {
+  const seenMessages = [];
```

Comment:
Let's rely on `Set` (functionality-wise it feels as better choice, also performance wise searching for a value in set is O(1) complexity, while in arrays it's O(n) - but ofc here it's just theoretical point, as we cannot deal here with large collections)